pkg/cloudprovider/vsphere/nodemanager.go

```diff
@@ -353,12 +356,15 @@ func (nm *NodeManager) DiscoverNode(nodeID string, searchBy cm.FindVM) error {
 				foundExternal = true
 				break
 			}
+
+			if !foundInternal && !foundExternal {
```

Comment:
Did we want to error out here? If error and dual stack is enabled, this won't attempt to try using the second ipfamily right?

Reply:
yeah, if neither is found, then we want to error out here. If error and dual-stack is enabled, the loop will exit here and won't attempt to use the second ipfamily.

Reply:
Ok cool